apps/playground/src/stories/hooks/useHover.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { useHover } from '../../../../../packages/react-hooks/use-hover';
+// import { useHover } from "@nanlabs/react-hooks";
+
+export const Example = () => {
+  const [isCbMemoBroken, setIsCbMemoBroken] = useState(false);
+  const [isTargetMemoBroken, setTargetMemoBroken] = useState(false);
```

Comment:
@boredcity could you use `isRefTarget` instead of `isTargetMemoBroken`?

Reply:
agreed, fixed